src/components/atoms/SNSIcon/index.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+import { StyleSocialIconCircle, StyleSocialIconContent } from "./styles"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome"
+import { faFacebookF, faInstagram, faMeetup } from "@fortawesome/free-brands-svg-icons"
+
+interface SNSIconInterface {
+    serviceName: string
+}
+
+function getSNSIcon(serviceName: string) {
+    switch (serviceName) {
+        case "facebook":
+            return faFacebookF
+        case "instagram":
+            return faInstagram
+        case "meetup":
+            return faMeetup
+    }
+}
+
+
+
+const SocialIcon: React.FC<SNSIconInterface> = (
+    props: SNSIconInterface
+) => {
+    return (
+        <StyleSocialIconCircle>
+            <StyleSocialIconContent href="#" target="_blank">
```

Comment:
`href` should come from props. And also, you need to add ` rel="noopener noreferrer" `.

RULE: When you link to a page on another site using the `target="_blank"`  you need to always use ` rel="noopener noreferrer" `

https://web.dev/external-anchors-use-rel-noopener/

Reply:
Is this correct?
`<StyleSocialIconContent href={props.url} target="_blank" rel="noopener noreferrer">`